packages/docs-reanimated/docs/advanced/useHandler.mdx

```diff
@@ -105,27 +99,10 @@ The hook returns a context that will be reused by event handlers and value that
 
 ## Example
 
```

Comment:
I'd also add a single sentence here that this example is easier to implement with `useScrollViewOffset`